server/app/services/program/ProgramService.java

```diff
@@ -782,7 +782,7 @@ public ErrorAnd<ProgramDefinition, CiviFormError> updateLocalization(
     ImmutableSet.Builder<CiviFormError> errorsBuilder = ImmutableSet.builder();
     validateProgramText(errorsBuilder, "display name", localizationUpdate.localizedDisplayName());
     validateProgramText(
-        errorsBuilder, "display description", localizationUpdate.localizedDisplayDescription());
+        errorsBuilder, "short display description", localizationUpdate.localizedShortDescription());
```

Comment:
short description is now required and our old description field (aka long description) is not.